### ISSUE TITLE
High-contrast mode: Comment buttons should use button text colour

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#7459]
This is a bug specific to [Windows high-contrast mode](https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/). cc @kbayliss who did the testing for this.

> Comment buttons should use the button text colour, and perhaps have a border too:

Current: `color: #007d7e`
Proposed: `color: ButtonText; border: 1px solid;`

![7459](https://user-images.githubusercontent.com/877585/129975545-aa9216c8-7551-46c9-8c9f-1d359f03c04a.png)

**Comment by RakeshPotnuru** [comment]
Hey @thibaudcolas, I am getting into open source. Can I work on this?

**Comment by Adyyousf** [comment]
hey @thibaudcolas I want to fix that.

**Comment by prakhar-asaiya** [comment]
Hey @thibaudcolas, can I take this issue?

**Comment by thibaudcolas** [comment]
@RakeshPotnuru Yes!If you haven’t already, have a look at our [Contributing to Wagtail](https://docs.wagtail.io/en/latest/contributing/index.html) guide first. For this specific issue, I expect the change is needed somewhere around here:

https://github.com/wagtail/wagtail/blob/c652e67c5e428998935e16a79b8255fd13966020/client/scss/components/_forms.scss#L470-L521

Note testing this properly will require using Windows high-contrast mode.

---

@prakhar-asaiya I think it would be best for Rakesh to look into this, but if you want there are plenty of other issues marked as "good first issue" on Wagtail.

**Comment by RakeshPotnuru** [comment]
@thibaudcolas 
Ok, I will work on this.

**Comment by RakeshPotnuru** [comment]
@thibaudcolas 
I am facing issues setting up the project in my local system, I asked in the slack support channel but no one responding.
Docs are not helping.
So can you please guide me?

**Comment by TUSHAR-VERMA-star** [comment]
Hey @thibaudcolas  I am getting into open source . can I take this issue ?

**Comment by thibaudcolas** [comment]
Hi @TUSHAR-VERMA-star, no, we already have a PR for this in #7648.

**Comment by lb-** [comment]
Merged in via #7648 & a0e55cedeeec3359a49ee82a3befd0a6d7e53d09
